### PR TITLE
Asic timelord docs

### PR DESCRIPTION
**Pull request by BrandtH22**

### CODE REVIEW

**Review by hoffmang9 — APPROVED** [review]
lgtm - only comment is "never overclock" might be too strong but I know why you chose it.

**Review by justinengland — DISMISSED** [review]

File: docs/getting-started/timelords.md [hunk 1]
```diff
@@ -6,10 +6,289 @@ slug: /timelords
...
+
+There are no specific requirements as timelords are a fastest wins process. This means that those with higher end hardware are more likely to generate Proofs of Time than those with lower end hardware.
+Currently, a modern gaming PC with 8 cores and 8 GB of RAM is recommended.
+
+### Can a Singe ASIC Compete with an ASIC Cluster?
+
+The nature of timelords is to create three VDF chains, one can create the chains themselves in parallel (i.e. one on each ASIC) but one cannot break down the VDFs themselves to parallelize them.
+This means that the ASIC cluster will always have an advantage but there are times when a single ASIC can reasonably compete. This almost always requires the block farming node to be closer in physical proximity to the single ASIC than to the ASIC cluster establishing a minor time advantage for the single ASIC
+
+### Can I Overclock the ASIC to Get More Performance or Higher IPS?
```
Comment (danieljperry) [inline docs/getting-started/timelords.md:280]: How about another question regarding the optimal voltage for running an ASIC?

File: docs/getting-started/timelords.md [hunk 2]
```diff
@@ -6,10 +6,289 @@ slug: /timelords
...
+
+## Timelord support
+
+Join Our [Discord](https://discord.gg/chia) and jump into the #support channel for support
+
+---
+
+## Timelord FAQ
+
+### What are the hardware requirements for running a Timelord?
```
[the remark this answers is not in the document]
Comment (BrandtH22) [reply docs/getting-started/timelords.md:270]: updated answer to reflect this and add more specificity

File: docs/getting-started/timelords.md (hunk 1 above)
Comment (BrandtH22) [reply docs/getting-started/timelords.md:280]: Question regarding voltage has eben added. Thank you for the suggestions!

**Review by danieljperry — DISMISSED** [review]

**Review by justinengland — APPROVED** [review]

**Pull request merged**